inst/report/summary_report.Rmd

```diff
@@ -380,8 +380,19 @@ cat(paste0("\n ",cat(paste0("* ", text2$prefix, "_", text2$source, "_"), sep = "
 
 ```{r, echo = FALSE, results = 'asis'}
 
+# Read in calibration options from calibration_options.yml, if that does not exist, 
+# use default options in options.yml
+
+calibration_files <- utils::unzip(output_zip, list = TRUE)
+
+if("info/calibration_options.yml" %in% calibration_files$Name) {
+  calibration_options <- unz(output_zip, "info/calibration_options.yml")
+  calibration_options <- yaml::read_yaml(calibration_options)
+} else {
+  calibration_options <- options[grep("calibration", names(options))]
```

Comment:
This code will not include the `calibrate_method` option. I would like to add that back into the report for completeness. Can you revise to add that back in?